aries-site/src/pages/foundation/accessibility.mdx

```diff
@@ -67,7 +67,7 @@ It is important to understand what is offered in terms of accessibility by the D
 | Accessibility consideration | Description | Grommet + HPE Theme | Design tokens only (separate lib) |
 |-----------------------------|-------------|----------------------|-----------------------------------|
 | **Core assets**             |             |                      |                                   |
-| Color                       | Contrast ratios | <Box align="center"> <CheckBox checked={true} /> </Box> | Review and apply: [Color palette](https://design-system.hpe.design/foundation/color?q=color) [Color usage](https://design-system.hpe.design/design-tokens/color-usage) [Color pairing](https://design-system.hpe.design/design-tokens/color-pairing) |
+| Color                       | Contrast ratios | <Box align="center"> <CheckBox checked={true} /> </Box> | Review and apply: [Color palette](https://design-system.hpe.design/foundation/color?q=color) [Color usage](https://design-system.hpe.design/design-tokens/color-usage) [Color pairing](https://design-system.hpe.design/foundation/color-pairing) |
```

Comment:
Internal links need only the path; the host portion of the URL is not necessary. Including the host portion treats the link as an external link and opens a new browser tab/window.

```suggestion
| Color                       | Contrast ratios | <Box align="center"> <CheckBox checked={true} /> </Box> | Review and apply: [Color palette](/foundation/color) [Color usage](/design-tokens/color-usage) [Color pairing](/foundation/color-pairing) |
```